MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+include tox.ini
+include LICENSE
+include MAKEFILE
+
+recursive-include tests *.sh *.ini
```

Comment:
Should you drop tox.ini from line 1 if you're recursively including `*.ini`? (It shouldn't hurt anything; it's just seemingly redundant)

Reply:
Hmm. I'll see about that. I guess I thought it was like `recursive-include <directory> <patterns...>` but I'm actually not sure.

Reply:
I believe you are correct. I'll go ahead and merge.